Assets/MRTK/SDK/Experimental/Joystick/JoystickController.cs

```diff
@@ -0,0 +1,162 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Joystick
+{
+    /// <summary>
+    /// Example script to demonstrate joystick control in sample scene
+    /// </summary>
+    public class JoystickController : MonoBehaviour
+    {
+        public GameObject ObjectToManipulate;
+        public TextMeshPro DebugText;
+
+        [SerializeField]
+        [Tooltip("The joystick mesh that gets rotated when this control is interacted with.")]
+        GameObject JoystickVisual = null;
```

Comment:
Please make sure to add [explicit access modifiers](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#access-modifiers) for clarity.